README.md

```diff
@@ -1,3 +1,50 @@
-# OSDC Blog
+<div align="center">
```

Comment:
Any specific reason why this is HTML and not standard markdown?

Reply:
Just so the logo can be centered - it looks better than way imo

Reply:
Fair enough. Thanks!